pkg/kafka/event_stream_writer.go

```diff
@@ -16,7 +16,7 @@ const (
 	WriteTimeout time.Duration = 5 * time.Second
 )
 
-//go:generate mockgen -source=event_stream.go -package=stream -destination=mock_event_stream.go
+//go:generate mockgen -source=event_stream_writer.go -package=kafka -destination=mock_event_stream_writer.go
```

Comment:
I think if this is not specific to the `Event` model. We shouldn't call it an `event_stream`.

I think just `kafka.Writer` would be fine.